### PR TITLE
WARN-1595 updates for rc qancode integration

### PR DESCRIPTION
**Pull request by caseylitton**
This is pointing at Jen qancode branch

### CODE REVIEW

File: qancode/comparisons.py [hunk 1]
```diff
@@ -194,6 +195,7 @@ def compare_data(self):
 
 class CompareScreenShots(URLComparison):
     def is_same(self, difference):
+        return False
```
Comment (jenjou) [inline qancode/comparisons.py]: Possible debug line?

File: qancode/qancode.py [hunk 2]
```diff
@@ -211,6 +213,10 @@ def find_differences(self,
                                                                          actions=actions,
                                                                          admin_only_actions=admin_only_actions,
                                                                          public_only_actions=public_only_actions)
+        item_types = ['/search/?type=Biosample&status=deleted&sort=accession']
+        click_paths = [None]
+        print(item_types)
```
Comment (jenjou) [inline qancode/qancode.py]: Debug line

File: qancode/comparisons.py (hunk 1 above)
Comment (caseylitton) [reply qancode/comparisons.py]: yes removed.

File: qancode/qancode.py (hunk 2 above)
Comment (caseylitton) [reply qancode/qancode.py]: cleaned up

**Pull request merged**